### ISSUE TITLE
Investigate Intels HDMI media stick

### ISSUE THREAD

**Issue by brock8503** [project-monolith/monolith-frontend#6]
This is a great device to reduce the overall setup cost of monolith http://www.intel.com/content/www/us/en/compute-stick/intel-compute-stick.html

Lets investigate if it would meet our needs.

**Comment by kd8drx** [comment]
Absolutely! At ~$110 for a lightweight linux version (which is all you need: just run a browser!), and another ~$250 or so for an LCD TV of decent size...
